Makefile

```diff
@@ -50,7 +50,7 @@ GIT_BRANCH := $(shell git rev-parse --abbrev-ref HEAD)
 
 # We don't want find to scan inside a bunch of directories, to accelerate the
 # 'make: Entering directory '/src/loki' phase.
-DONT_FIND := -name tools -prune -o -name vendor -prune -o -name .git -prune -o -name .cache -prune -o -name .pkg -prune -o
+DONT_FIND := -name tools -prune -o -name vendor -prune -o -name operator -prune -o -name .git -prune -o -name .cache -prune -o -name .pkg -prune -o
```

Comment:
other there any implications here for removing the operator directory? like do they lose linting?

Reply:
they do have their own `.golangci-lint.yml`, and the concern was we may need to make formatting decisions based on ours that won't agree with their. that being said this may be a YAGNI, so I'll remove it for now and wait for it to be a problem.